README.md

```diff
@@ -74,37 +78,28 @@ Then, you can create propagated span object by decoding these items.
 SpanContextPtr parent_span = createSpanContext(parent);
 ```
 
-#### Create workload segment
-
-Create segment for current workload.
-
-```cpp
-SegmentConfig seg_config(service_name, instance_name);
-SegmentContextPtr current_segment = createSegmentContext(seg_config);
-```
-
 #### Create span
 
 First, you must create root span to trace current workload.
 
 ```cpp
-CurrentSegmentSpanPtr current_span = current_segment->createEntrySpan();
+TracingSpanPtr tracing_span = tracing_context->createEntrySpan();
```

Comment:
I find `TracingContextPtr tracing_context = tracer->newContext();` API is at the end of the document only. Could you fix it?

Reply:
fixed